bazel/integration/tests/angular-cli/test/wdio.conf.js

```diff
@@ -0,0 +1,38 @@
+exports.config = {
+  specs: [__dirname + '/specs/**/*.ts'],
+  capabilities: [
+    {
+      browserName: 'chrome',
+      'goog:chromeOptions': {
+        // Allows us to use Chromium provided by Bazel.
+        binary: process.env.CHROME_BIN,
+        // Needed so that the tests can run headless within Bazel and its sandbox.
+        args: ['--headless', '--no-sandbox', '--disable-gpu'],
+      },
+    },
+  ],
+  services: [
+    [
+      'chromedriver',
+      {
+        // Allows us to use the chromedriver path provided by Bazel.
+        chromedriverCustomPath: process.env.CHROMEDRIVER_PATH,
```

Comment:
Note: This config is auto-generated by the schematics. I just added the custom chromedriver path environment variable + the custom chrome options above.